packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -206,6 +206,10 @@ async function runAxeTestsForBlockEditor() {
 			'aria-required-parent',
 			'frame-title',
 		],
+		disabledFrames: [
+			// Ignore block previews.
+			'[aria-hidden="true"]',
```

Comment:
The iframe in question is added by `react-resize-aware` as part of the `useResizeObserver` hook and has `aria-hidden="true"` on it, but as a general rule it seems like axe shouldn't be trying to run tests on the contents of iframes that are aria-hidden.

Reply:
Added a comment to explain this.